README.md

```diff
@@ -1 +1,98 @@
-# Things3 Low-Level Python API to the Database
\ No newline at end of file
+Get your Things app data into a Python dict.
```

Comment:
I suggest some introduction with some background and motivation

Reply:
I enjoy examples and TLDR first. It seems to me most skim READMEs and only dive deeper later. What do you think?

Reply:
Agree to jump quickly to the code, at the same time, *a bit* of context for a reader is a must.

Reply:
What more context do you think is needed? What was missing for you?

Reply:
I'll think about / add something later ;)